fabric-fluid-v1/src/main/java/net/fabricmc/fabric/Action.java

```diff
@@ -0,0 +1,24 @@
+package net.fabricmc.fabric;
+
+/**
+ * describes the behavior of a fluid transaction
+ */
+public enum Action {
+	/**
+	 * actually mutate and update the container's state.
+	 */
+	PERFORM,
+
+	/**
+	 * simulate the transaction, do not mutate the state of the container itself.
+	 */
+	SIMULATE;
+
+	public boolean simulate() {
+		return this == SIMULATE;
+	}
+
+	public boolean perform() {
```

Comment:
isPerforming

Reply:
shouldPerform sounds better, does that work? we should have yarn for fapi /s

Reply:
that works